certbot/cli.py

```diff
@@ -1103,8 +1104,9 @@ def _create_subparsers(helpful):
     helpful.add(
         None, "--user-agent", default=None,
         help="Set a custom user agent string for the client. User agent strings allow "
-             "the CA to collect high level statistics about success rates by OS and "
-             "plugin. If you wish to hide your server OS version from the Let's "
+             "the CA to collect high level statistics about success rates by OS, "
+             "plugin and use case, and to know when to deprecate support for past Python "
+             "versions. If you wish to hide this information from the Let's "
```

Comment:
This is worded kind of inaccurately.

The UA string does allow the CA to collect usage information, but it's the client developers who make decisions about things like which python versions to support (not the CAs).